src/Controller/Backend/BackendBase.php

```diff
@@ -71,6 +71,7 @@ public function before(Request $request, Application $app, $roleRoute = null)
         // Sanity checks for doubles in in contenttypes. This has to be done
         // here, because the 'translator' classes need to be initialised.
         $app['config']->checkConfig();
+        $app['config']->checkTaxonomy();
```

Comment:
OK … soo looking at it, and thinking about it … probably best for now to put that at the end of `checkConfig()` as that is one of the things on my :fire: list for the 3.2 exception/request work.

Short version is that we're checking on every request the configuration of the configuration  … ConfigCeption … that is already cached … if it is cached, then we know it's already passed checks.